skimage/data/__init__.py

```diff
@@ -648,6 +650,32 @@ def coins():
     return _load("data/coins.png")
 
 
+def kidney():
+    """Microscopy image of kidney tissue.
+    See `kidney-tissue-fluorescence.tif` entry at
+    https://gitlab.com/scikit-image/data/-/blob/master/README.md#data
+
+    Returns
+    -------
+    kidney : (16, 512, 512, 3) uint16 ndarray
+        Kidney image.
+    """
+    return _load("data/kidney.tif")
+
+
+def lily():
+    """Microscopy image of a lily of the valley.
+    See `lily-of-the-valley-fluorescence.tif` entry at
+    https://gitlab.com/scikit-image/data/-/blob/master/README.md#data
+
+    Returns
+    -------
+    lily : (922, 922, 4) uint16 ndarray
```

Comment:
Is this shape acceptable for our datasets?

I Feel like the output shape should be `(4, 922, 922)`, in C-contiguous order

Reply:
4 is the number of colour channels: `lily` is a 2D multichannel image (https://scikit-image.org/docs/dev/auto_examples/applications/plot_3d_image_processing.html#explore-3d-images-of-cells). I'll add it in the docstring.

Reply:
Got it, yeah that wasn't clear to me. I was assuming `x, y, z`